.github/workflows/revert.yml

```diff
@@ -56,15 +50,6 @@ jobs:
         run: |
           bash app-frontend/.github/scripts/revert.sh \
```

Comment:
did you mean to remove  `app-frontend/` prefix here since you added working directory?

Reply:
yes, good catch